optimize/pom.xml

```diff
@@ -26,17 +26,13 @@
     <surefire-plugin.version>3.4.0</surefire-plugin.version>
 
     <!-- database config -->
-    <elasticsearch.version>7.17.23</elasticsearch.version>
-    <opensearch.version>2.10.1</opensearch.version>
+    <elasticsearch.client.version>7.17.23</elasticsearch.client.version>
```

Comment:
Out of curiosity: shouldn't it be 8.13.0? 🤔

Reply:
This is just the client version, that doesn't get bumped up until Ivan's changes. This PR is just changing the support for the underlying DB